src/binaryen.h

```diff
@@ -41,6 +41,10 @@ class BinaryenEngine : public WasmEngine {
 
 private:
   void verifyContract(wasm::Module & module);
+
+  /// Parses and loads a Wasm module.
+  /// Don't ask, Module has no copy constructor, hence the reference.
+  void loadModule(std::vector<uint8_t> const& code, wasm::Module & module);
```

Comment:
Perhaps we could move this to a virtual method so that any abstract VM should have to implement it?

Reply:
It would be super abstract 😉